frontend/src/containers/LoginPage/LoginPage.js

```diff
@@ -1,82 +1,30 @@
 import React, { Component } from 'react';
```

Comment:
It looks like you have a LoginPage in both `components` and `containers`. I think we were hoping to standardize around using `containers` for pages that hold other components, but either way, we probably just want one of them.

Reply:
I think this is actually why CI is failing right now, as something is referencing the `containers` version, where the import paths would need to be fixed (since the other components aren't in the same directory).